integration_tests/fixtures.py

```diff
@@ -87,7 +87,7 @@ def wait_for_presto_workers(host, port, timeout=180):
         time.sleep(1)
 
 
-def wait_for_presto_coordinator(stream, timeout=180):
+def wait_for_presto_coordinator(stream, timeout=360):
```

Comment:
I could not pull image and run container in time otherwise. I guess I might not be alone with this issue (slow internet?)